Rainbow/rainbow.css

```diff
@@ -0,0 +1,24 @@
+.rainbow {
+  background: -webkit-linear-gradient(left, red, orange , yellow, green, cyan, blue, violet);
+  background: -o-linear-gradient(right,red,  orange, yellow, green, cyan, blue, violet);
+  background: -moz-linear-gradient(right,red,  orange, yellow, green, cyan, blue, violet);
+  background: linear-gradient(to right,red,  orange , yellow, green, cyan, blue, violet);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  display: inline-block;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.shadow {
+  position: absolute;
+  left: 13px;
```

Comment:
Pixel positioning will only fit properly on h2.rainbow, won't it? What if the class is set on a h1, h5 or p? Do we want to support arbitrary text size?

Reply:
Suprisingly no! I have tested it with the different headers and paragraph tags and they seem to work. Since both the text and the "shadow" are in the same container the `absolute` -tag positions them properly to each other. As long as they both have the same tag that is.

Reply:
The `left` seems to not work correctly on small font sizes, e.g. for a `p` it looks like this: 

![image](https://user-images.githubusercontent.com/2099030/61189161-c5c0fb80-a689-11e9-91fa-afcd90e3013f.png)

Perhaps we could set it to a fraction of the font size, instead?

Reply:
Dang it. Then we need to do that. 
I am not happy with the shadows in their current state. I kinda want to remove them outright

Reply:
That would not be true to the original form, but it might be acceptable to do so now and add shadows later.